apps/ltn/src/colors.rs

```diff
@@ -16,7 +16,7 @@ lazy_static::lazy_static! {
         Color::hex("#ffed6f"),
     ];
 
-    pub static ref PLAN_ROUTE_BEFORE: Color = Color::RED;
+    pub static ref PLAN_ROUTE_BEFORE: Color = Color::PURPLE;
```

Comment:
Red means "bad" and really clashes with the new view, just switching to some other arbitrary color